src/cljs/main/broadfcui/page/workspace/summary/acl_editor.cljs

```diff
@@ -10,11 +10,11 @@
     [broadfcui.utils :as utils]
     ))
 
-
+; Note that render-acl-content below requires that these values be in descending order
```

Comment:
Wow, I didn't even notice this copy of the access levels here. This is duplicated from `WorkspaceACL` in rawls. Just noting here that this is tech debt (and also that `PROJECT_OWNER` is missing in case that matters).